src/util/irep_ids.def

```diff
@@ -662,8 +662,11 @@ IREP_ID_TWO(overflow_shl, overflow-shl)
 IREP_ID_TWO(C_no_initialization_required, #no_initialization_required)
 IREP_ID_TWO(overlay_class, java::com.diffblue.OverlayClassImplementation)
 IREP_ID_TWO(overlay_method, java::com.diffblue.OverlayMethodImplementation)
+IREP_ID_ONE(is_annotation)
 IREP_ID_TWO(C_annotations, #annotations)
 IREP_ID_ONE(final)
+IREP_ID_ONE(synthetic)
+IREP_ID_ONE(interface)
```

Comment:
Even if there isn't a feature we know about yet for which we _have to_ distinguish between between interfaces and abstract classes, I think it is worth preserving this information from the class files. Otherwise we may have to return to this later to separate one flag into two.

Reply:
We currently tag interface types with `ID_abstract`. If we want to change this, the distinction needs to be made in all places where we use `ID_abstract`.

Reply:
I agree with Tom and since is `ID_abstract` field is left unchanged by this PR I think nothing needs updating (I suppose interfaces are marked as abstract in the bytecode).

Reply:
Agree with @thk123 -- interfaces are abstract, not all abstract things are interfaces, so I think this does the right thing already.